src/quo/components/settings/data_item/view.cljs

```diff
@@ -58,10 +59,9 @@
       (i18n/label :t/days)])])
 
 (defn- left-side
-  "The description can either be given as a string `description` or a component `custom-subtitle`"
-  [{:keys [theme title status size blur? description custom-subtitle icon subtitle label icon-color
-           customization-color network-image
-           emoji]
+  "The description can either be given as a string `subtitle-type` or a component `custom-subtitle`"
```

Comment:
Probably `subtitle-type` should be changed to `subtitle` here, because `subtitle-type` is not string, if I understood the component right.